### PR TITLE
Single project

### PR DESCRIPTION
**Pull request by nhulz**
Mostly just working on the one single-project page. Some editing done to get the taxonomies to work correctly.

### CODE REVIEW

File: sass/project.sass [hunk 1]
```diff
@@ -22,45 +26,118 @@
...
   text-transform: uppercase
   font-size: 4rem
   letter-spacing: 20px
 
-.project__subtitle
+.project__details,
+.project__subtitle,
+.project__content-title,
+.project__small-header,
+p
```
Comment (alfred) [inline sass/project.sass]: Setting the element styles by using the `p` selector is gonna bite you in the ass later when you need to override it.

This composed rule might make more sense to be on the `.project` class or whatever the wrapper is called

File: sass/project.sass [hunk 2]
```diff
@@ -22,45 +26,118 @@
...
+  @media screen and (min-width: 768px)
+    +span-columns(6 of 12)
+  img
+    +span-columns(12)
+
+.layout__col-6--left
+  +row
+  +span-columns(12)
+  @media screen and (min-width: 768px)
+    +span-columns(8 of 12)
```
Comment (alfred) [inline sass/project.sass]: This line should probably be `span-columns(6 of 12)`

File: sass/project.sass [hunk 3]
```diff
@@ -73,6 +150,36 @@
...
+  @media screen and (min-width: 768px)
+    margin-bottom: 10rem
+
+.layout__background-color
+  height: 40rem
+  width: 100%
+  position: absolute
+  top: 0
+  left: 0
+  z-index: -1
```
Comment (alfred) [inline sass/project.sass:215]: This class is fun cause its named for `background-color` but there is no mention of color at all

**Review by alfred — CHANGES_REQUESTED** [review]
1. What is with the inconsistent ass spacing in the html?
2. Remove the commented out pieces of broken code
3. Remove the `| print_r` lines from PHP
1. I also question the use of `rem` but it is you're an adult and can do what you want

File: sass/project.sass (hunk 3 above)
Comment (nhulz) [reply sass/project.sass:215]: Idk if it's obvious but the color is set inline by an ACF

**Pull request merged**